ui/core/components/inputs/consumables.ts

```diff
@@ -519,81 +502,69 @@ export const PetScrollOfStrengthV = makeBooleanConsumeInput({
 };
-export const RunicManaPotion = {
-	actionId: ActionId.fromItemId(33448),
-	value: Potions.RunicManaPotion,
+
+export const VolcanicPotion = {
+	actionId: ActionId.fromItemId(58091),
+	value: Potions.VolcanicPotion,
 };
-export const RunicManaInjector = {
-	actionId: ActionId.fromItemId(42545),
-	value: Potions.RunicManaInjector,
+
+export const EarthenPotion = {
+	actionId: ActionId.fromItemId(58090),
+	value: Potions.EarthenPotion,
 };
-export const IndestructiblePotion = {
-	actionId: ActionId.fromItemId(40093),
-	value: Potions.IndestructiblePotion,
+
+export const MightyRejuvenationPotion = {
+	actionId: ActionId.fromItemId(57193),
+	value: Potions.MightyRejuvenationPotion,
 };
-export const PotionOfSpeed = { actionId: ActionId.fromItemId(40211), value: Potions.PotionOfSpeed };
-export const PotionOfWildMagic = {
-	actionId: ActionId.fromItemId(40212),
-	value: Potions.PotionOfWildMagic,
+
+export const MythicalHealingPotion = {
+	actionId: ActionId.fromItemId(57191),
+	value: Potions.MythicalHealingPotion,
 };
 
-export const DestructionPotion = {
-	actionId: ActionId.fromItemId(22839),
-	value: Potions.DestructionPotion,
+export const MythicalManaPotion = {
+	actionId: ActionId.fromItemId(57192),
+	value: Potions.MythicalManaPotion,
 };
+export const PotionOfSpeed = { actionId: ActionId.fromItemId(40211), value: Potions.PotionOfSpeed };
 export const HastePotion = { actionId: ActionId.fromItemId(22838), value: Potions.HastePotion };
 export const MightyRagePotion = {
 	actionId: ActionId.fromItemId(13442),
 	value: Potions.MightyRagePotion,
 };
-export const SuperManaPotion = {
-	actionId: ActionId.fromItemId(22832),
-	value: Potions.SuperManaPotion,
-};
-export const FelManaPotion = { actionId: ActionId.fromItemId(31677), value: Potions.FelManaPotion };
-export const InsaneStrengthPotion = {
-	actionId: ActionId.fromItemId(22828),
-	value: Potions.InsaneStrengthPotion,
-};
-export const IronshieldPotion = {
-	actionId: ActionId.fromItemId(22849),
-	value: Potions.IronshieldPotion,
-};
-export const HeroicPotion = { actionId: ActionId.fromItemId(22837), value: Potions.HeroicPotion };
 
 export const POTIONS_CONFIG = [
-	{ config: RunicHealingPotion, stats: [Stat.StatStamina] },
-	{ config: RunicHealingInjector, stats: [Stat.StatStamina] },
-	{ config: RunicManaPotion, stats: [Stat.StatIntellect] },
-	{ config: RunicManaInjector, stats: [Stat.StatIntellect] },
-	{ config: IndestructiblePotion, stats: [Stat.StatArmor] },
-	{ config: InsaneStrengthPotion, stats: [Stat.StatStrength] },
-	{ config: HeroicPotion, stats: [Stat.StatStamina] },
+	{ config: GolembloodPotion, stats: [Stat.StatStrength] },
+	{ config: PotionOfTheTolvir, stats: [Stat.StatAgility] },
+	{ config: PotionOfConcentration, stats: [Stat.StatMana] },
+	{ config: VolcanicPotion, stats: [Stat.StatIntellect] },
+	{ config: EarthenPotion, stats: [Stat.StatArmor] },
+	{ config: MightyRejuvenationPotion, stats: [Stat.StatMana, Stat.StatHealth] },
```

Comment:
I think we may have to add Mana as an EP stat to the sims that have mana as its no longer connected to intellect right?

Reply:
It's only changed in mop. Cata still increases mana by intellect, I dont think that has changed going from wotlk to cata

Reply:
In that case the stat that this potion is tied to should be Intellect as we're already including it in sims EP values

Reply:
Fixed